packages/main/src/StepInput.hbs

```diff
@@ -52,6 +54,7 @@
 	</ui5-input>
 
 	<!-- Increment Icon -->
+	{{#if hasNotReadOnly}}

```

Comment:
you can use unless readonly, instead of if hasNotReadOnly

Reply:
done